python/raf/__init__.py

```diff
@@ -3,7 +3,9 @@
 
 """RAF is Not MXNet, it's MXNet 3.0."""
 
-__version__ = "0.0.2.dev"
+from .version import __version__ as __version__
+from .version import __full_version__ as __full_version__
+from .version import __gitrev__ as __gitrev__
```

Comment:
This will result in errors if developers don't run `setup.py` so `version.py` isn't generated. Better to have a guard:

```python
try:
    from .version import __version__ as __version__
    from .version import __full_version__ as __full_version__
    from .version import __gitrev__ as __gitrev__
except:  # pylint: disable=bare-except
    __version__ = "dev"
    __full_version__ = "dev"
    __gitrev__ = "unkonwn"
```

Reply:
I see, thanks